### PR TITLE
PR 1204 Fixes

### PR DESCRIPTION
**Pull request by christophermay07**
Fixes implemented to allow CI to complete on all servers.

### CODE REVIEW

**Comment by christophermay07** [comment]
@TabassumJafri assigned for review

**Comment by christophermay07** [comment]
@mhpnguyen assigned for review

File: Product/Production/Adapters/AdminDistribution_a0/src/main/java/gov/hhs/fha/nhinc/admindistribution/adapter/AdapterAdministrativeDistributionSecured.java [hunk 1]
```diff
@@ -33,15 +34,13 @@
 import javax.jws.WebService;
 import javax.xml.ws.BindingType;
```
Comment (TabassumJafri) [inline Product/Production/Adapters/AdminDistribution_a0/src/main/java/gov/hhs/fha/nhinc/admindistribution/adapter/AdapterAdministrativeDistributionSecured.java:32]: WebService import is unused.

File: Product/Production/Services/DirectCore/src/test/java/gov/hhs/fha/nhinc/direct/xdr/DirectClientFactoryTest.java [hunk 2]
```diff
@@ -49,7 +49,6 @@
      */
     @BeforeClass
     public static void setUpClass() {
```
Comment (mhpnguyen) [inline Product/Production/Services/DirectCore/src/test/java/gov/hhs/fha/nhinc/direct/xdr/DirectClientFactoryTest.java]: remove if it doesn't do anything

File: Product/Production/Gateway/CONNECTGatewayWeb/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/PatientCorrelationServiceSecured.java [hunk 3]
```diff
@@ -43,8 +41,7 @@
  *
  * @author Sai Valluripalli
  */
-@WebService
-@BindingType(value = SOAPBinding.SOAP12HTTP_BINDING)
+@BindingType(value = javax.xml.ws.soap.SOAPBinding.SOAP12HTTP_BINDING)
```
Comment (TabassumJafri) [inline Product/Production/Gateway/CONNECTGatewayWeb/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/PatientCorrelationServiceSecured.java]: We already have a javax.xml.ws.soap.SOAPBinding import. Why we need to define a fully qualified class name in @BindingType annotation.

File: Product/Production/Common/CONNECTCommonWeb/src/main/java/gov/hhs/fha/nhinc/connectmgr/uddi/UDDIUpdateManager.java [hunk 4]
```diff
@@ -52,7 +52,6 @@ public UDDIUpdateManager() {
         }
     }
 
-    @WebMethod
```
Comment (TabassumJafri) [inline Product/Production/Common/CONNECTCommonWeb/src/main/java/gov/hhs/fha/nhinc/connectmgr/uddi/UDDIUpdateManager.java:55]: Unused WebMethod import

File: Product/Production/Gateway/CONNECTGatewayWeb/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/PatientCorrelationServiceUnsecured.java [hunk 5]
```diff
@@ -44,8 +42,7 @@
  *
  * @author jhoppesc
  */
-@WebService
-@BindingType(value = SOAPBinding.SOAP12HTTP_BINDING)
+@BindingType(value = javax.xml.ws.soap.SOAPBinding.SOAP12HTTP_BINDING)
```
Comment (TabassumJafri) [inline Product/Production/Gateway/CONNECTGatewayWeb/src/main/java/gov/hhs/fha/nhinc/patientcorrelation/nhinc/PatientCorrelationServiceUnsecured.java]: same comment as above.

**Comment by TabassumJafri** [comment]
Reviewed and good with merge.

**Pull request merged**